src/Zikula/HookBundle/Twig/Extension/HookExtension.php

```diff
@@ -106,8 +106,11 @@ public function createRouteUrl(string $routeName, array $routeParameters = [], s
      */
     public function notifyFilters(string $content, string $filterEventName)
     {
-        $hook = new FilterHook($content);
+        $hook = $this->hookDispatcher->dispatch($filterEventName, new FilterHook($content));
```

Comment:
Shouldn't the argument order be swapped?

Reply:
The hook dispatcher still takes the arguments in this order for the time being. [It swaps them when dispatching through the eventDispatcher](https://github.com/zikula/core/blob/0c4de2309938827f816cb9e51e50b5b6443decb9/src/Zikula/HookBundle/Dispatcher/HookDispatcher.php#L51). We could consider changing this later.